### PR TITLE
team page, update header, login, and signup page styles

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
add team to about page
## How to Test
- [ ] Visit /work/about
- [ ] Verify our photos/profile/links are correct

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/work/TeamCard/TeamCard.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  avatar: string;
+  handle?: string;
+  social: {
+    iconClass: string;
+    link: string;
+  }[];
+}
+
+const TEAM_CARD_STYLES = {
+  height: '310px',
```
Comment (kcvan) [inline src/components/work/TeamCard/TeamCard.tsx]: hard height because height can vary based on items inside the card

**Comment by tc** [comment]
the twitter links don't need `@`
